sycl/doc/GetStartedGuide.md

```diff
@@ -164,11 +164,15 @@ which contains all the symbols required.
 To run DPC++ applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
+To run DPC++ applications on Level Zero devices, Level Zero implementation(s)
```

Comment:
Please add reference to Level Zero documentation at the bottom of the page and link to this reference here.